Package.swift

```diff
@@ -47,54 +48,65 @@ let isLinux = false
-    .library(name: "SwiftGRPC", targets: ["SwiftGRPC"]),
-    .library(name: "SwiftGRPCNIO", targets: ["SwiftGRPCNIO"]),
-  ],
-  dependencies: packageDependencies,
-  targets: [
-    .target(name: "SwiftGRPC",
-            dependencies: ["CgRPC", "SwiftProtobuf"]),
-    .target(name: "SwiftGRPCNIO",
-            dependencies: [
-              "NIOFoundationCompat",
-              "NIOHTTP1",
-              "NIOHTTP2",
-              "SwiftProtobuf"]),
-    .target(name: "CgRPC",
-            dependencies: cGRPCDependencies),
-    .target(name: "RootsEncoder"),
-    .target(name: "protoc-gen-swiftgrpc",
-            dependencies: [
-              "SwiftProtobuf",
-              "SwiftProtobufPluginLibrary",
-              "protoc-gen-swift"]),
-    .target(name: "BoringSSL"),
-    .target(name: "Echo",
-            dependencies: [
-              "SwiftGRPC",
-              "SwiftProtobuf",
-              "Commander"],
-            path: "Sources/Examples/Echo"),
-    .target(name: "EchoNIO",
-            dependencies: [
-              "SwiftGRPCNIO",
-              "SwiftProtobuf",
-              "Commander"],
-            path: "Sources/Examples/EchoNIO"),
-    .target(name: "Simple",
-            dependencies: ["SwiftGRPC", "Commander"],
-            path: "Sources/Examples/Simple"),
-    .testTarget(name: "SwiftGRPCTests", dependencies: ["SwiftGRPC"]),
-    .testTarget(name: "SwiftGRPCNIOTests", dependencies: ["SwiftGRPC", "SwiftGRPCNIO"])
-  ],
-  swiftLanguageVersions: [.v4, .v4_2, .version("5")],
-  cLanguageStandard: .gnu11,
-  cxxLanguageStandard: .cxx11)
+    name: "SwiftGRPC",
+    products: [
+        .library(name: "SwiftGRPC", targets: ["SwiftGRPC"])
+    ],
+    dependencies: packageDependencies,
+    targets: [
+        .target(name: "SwiftGRPC",
+                dependencies: ["CgRPC", "SwiftProtobuf"]),
+        .target(name: "CgRPC",
+                dependencies: cGRPCDependencies,
+                cSettings: [
+                    .headerSearchPath("../BoringSSL/include", .when(platforms: [.iOS, .macOS, .tvOS, .watchOS])),
+                    .unsafeFlags(["-Wno-module-import-in-extern-c"])],
+                linkerSettings: [.linkedLibrary("z")]),
+        .target(name: "RootsEncoder"),
+        .target(name: "protoc-gen-swiftgrpc",
+                dependencies: [
+                    "SwiftProtobuf",
+                    "SwiftProtobufPluginLibrary",
+                    "protoc-gen-swift"]),
+        .target(name: "BoringSSL"),
+        .target(name: "Echo",
+                dependencies: [
+                    "SwiftGRPC",
+                    "SwiftProtobuf",
+                    "Commander"],
+                path: "Sources/Examples/Echo"),
+        .target(name: "Simple",
+                dependencies: ["SwiftGRPC", "Commander"],
+                path: "Sources/Examples/Simple"),
+        .testTarget(name: "SwiftGRPCTests", dependencies: ["SwiftGRPC"])
+    ],
+    swiftLanguageVersions: [.v4, .v4_2, .version("5")],
```

Comment:
nit: `.v5` since we now have `swift-tools-version:5.0`

Reply:
Done.